chromium_src/chrome/browser/profiles/profile.cc

```diff
@@ -3,7 +3,9 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-#include "../../../../../../chrome/browser/profiles/profile.cc"
+#include "../../../../../../chrome/browser/profiles/profile.cc"  // NOLINT
+
+#include "brave/chromium_src/chrome/browser/profiles/profile.h"
```

Comment:
fyi - this lint check is now disabled so we no longer have to add these here

Reply:
disabled in https://github.com/brave/brave-core/pull/2647/commits/3bc4178c9ee409e6126e0f5055bd86afe5f687e9

Reply:
addressed in cf2d8e5629839c23a8f7db0e95b77b857a0ff12a